templates/CRM/Admin/Page/APIExplorer.tpl

```diff
@@ -222,13 +222,13 @@
 <div class="crm-block crm-content-block">
 <div id="mainTabContainer">
   <ul>
-    <li class="ui-corner-all" title="GUI to build and execute API calls">
+    <li class="ui-corner-all" title="{ts}GUI to build and execute API calls{/ts}">
       <a href="#explorer-tab"><i class="crm-i fa-search" aria-hidden="true"></i> {ts}Explorer{/ts}</a>
     </li>
-    <li class="ui-corner-all" title="Auto-generated examples from the test suite">
+    <li class="ui-corner-all" title="{ts}Auto-generated examples from the test suite{/ts}">
       <a href="#examples-tab"><i class="crm-i fa-book" aria-hidden="true"></i> {ts}Examples{/ts}</a>
     </li>
-    <li class="ui-corner-all" title="API source-code and code-level documentation">
+    <li class="ui-corner-all" title="{ts}API source-code and code-level documentation{/ts}">
```

Comment:
Because APIv3 has already been superseded by v4, I don't think there's any need to burden translators with these strings. Either leave as-is untranslated (this is a developer tool, and code is always written in English), or remove them.

Reply:
Fair enough, reverted.